src/docs/content/observability/oci-healthcheck.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "OCI Health Check"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 0
+tags:
+  - Monitoring
+  - oci_healthcheck
+---
+
+{{% alert style="warning" icon="warning" %}}
+Note that this is **OPTIONAL**.  
+{{% /alert %}}
```

Comment:
I think we can omit the optional banner for each of these. This has been used when introducing topics mid-stream which could be skipped without impacting deployment